GoogleForCreators/web-stories-wp
# Editor: remove duplicate icons

## Context



See issue #8555 for details.
This PR kept some of the icons because currently there is no alternatives for following icons in `design-system/icons`.

story-editor/src/icons/index.js:
```
export { ReactComponent as TurningLine } from './turning_line.svg';
export { ReactComponent as Union } from './union.svg';
export { ReactComponent as Muted } from './volume_off.svg';
```

## Summary



## Relevant Technical Choices



## To-do



## User-facing changes

 Updates icons used for: 
- edit/remove media 
- selected font in font picker dropdown 
- Search icon in font picker 

## Testing Instructions

1. Hover over media in media panel, see 3 dot circle button - this is the icon that was updated. 

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8555
